File: app_rest/app/model/class/coordinate.php
<?php

class Coordinate {
	public $Latitude;
	public $Longitude;
	public $Altitude;

	public function __construct($latitude, $longitude, $altitude) {
		$this->Latitude = (double) $latitude;
		$this->Longitude = (double) $longitude;
		$this->Altitude = (double) $altitude;

	}
}

?>

File: app_socket/app_socket/GaiaWatcher/Classes/Coordinate.cs
using System;
using System.Collections.Concurrent;
using System.Collections.Generic;
using System.Linq;
using System.Net.Sockets;
using System.Text;

namespace GaiaWatcher {
    public class Coordinate {
        public double latitude {
            get;
            set;
        }
        public double longitude {
            get;
            set;
        }
        public double altitude {
            get;
            set;
        }
        public int course {
            get;
            set;
        }
    }
}
